src/chatkit.ts

```diff
@@ -11,17 +11,26 @@ import {
   TokenWithExpiry,
 } from "@pusher/platform-node"
 
-import { getCurrentTimeInSeconds } from "./utils"
+import {
+  getCurrentTimeInSeconds,
+  validateProperties,
+} from "./utils"
 import packageJSON from "../package.json"
+import * as t from 'io-ts'
+import "unknown-ts"
+
+const NonEmptyString = t.refinement(t.string, s => s.length != 0, "NonEmptyString");
```

Comment:
Though it wouldn't make a difference here, triple equals is always preferred (i.e. !==)